src/Integration.UnitTests/Exclusions/AnalyzableFileIndicatorTests.cs

```diff
@@ -142,6 +142,28 @@ public void ShouldAnalyze_HasInclusions_NoExclusions_ReturnsIfIncluded(bool incl
             result.Should().Be(expectedResult);
         }
 
+        [TestMethod]
+        public void ShouldAnalyze_HasBackSlash_ReplacesWithForwardSlash()
```

Comment:
It would help if the test name made it clear what had the backslash (i.e. the file name, as opposed to the pattern).